app-emulation/crun/crun-0.10.6.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+inherit python-any-r1
+
+DESCRIPTION="A fast and low-memory footprint OCI Container Runtime fully written in C"
+HOMEPAGE="https://github.com/containers/crun"
+SRC_URI="https://github.com/containers/${PN}/releases/download/${PV}/${P}.tar.gz"
```

Comment:
Why use releases instead of archive here?

Reply:
Giuseppe maintains a package called libocispec in a seperate repo, but includes it here as a submodule, as part of `make dist` it's include in the source release. On a side note I have yet to see another package use libocispec.